.travis.yml

```diff
@@ -151,6 +165,15 @@ before_install:
   - ulimit -a
   - mkdir builds
   - cd builds
+  - |
+    if [ "${USE_DEBUG}" == "1" ];
+    then
+        # Work in our own virtualenv to isolate from travis-ci packages.
+        travis_retry pip install -U virtualenv
```

Comment:
I'll remove it. I'm not sure if using ``language: generic`` was the best decision, the problem is that the version of ``python3-dbg`` is going to depend on the distro used (3.5.1-3) for Xenial. I didn't want to specify a specific Python version as this could confuse others editing it in the future. *i.e. why specify python3.X if pytho3-dbg uses a different version. 

Do you think using ``language: generic`` is clearer?

Reply:
You can tell `apt` to install `python3.5-dbg` and use `python3.5` in the virtualenv command to specify the version.

Reply:
Ok, I wanted to avoid hard coding a version so I was playing around with setting an environment variable. But that makes the implementation much more complicated than it needs to be. I'll add the apt packages manually.